context/amf_ue.go

```diff
@@ -69,7 +69,7 @@ type AmfUe struct {
 	// Mutex sync.Mutex `json:"mutex,omitempty" yaml:"mutex" bson:"mutex,omitempty"`
 	Mutex sync.Mutex `json:"-"`
 	/* the AMF which serving this AmfUe now */
-	servingAMF *AMFContext `json:"servingAMF,omitempty"` // never nil
+	servingAMF *AMFContext // never nil
```

Comment:
Why not doing the following (with the corresponding changes in the respective files)?
```suggestion
	ServingAMF *AMFContext `json:"servingAMF,omitempty"` // never nil
```

Reply:
I tried doing it for this particular case, but it unfortunately clashes with a method named exactly like that. I could change the name of the attribute further, or change the method name, but this will be more involved.

Reply:
I pushed a new commit that does this rename and also gets rid of the method for a direct attribute access instead.